### DISCUSSION TITLE
Looking for team members for project

### DISCUSSION THREAD

**Discussion by stan6** [SOEN345-WINTER2024/discussion#8] [Q&A]
Please post here if you are looking for members for the project.

In https://github.com/orgs/SOEN345-WINTER2024/discussions/4, there are multiple groups only one student as well.

**Comment by MinhDuc1711** [comment]
Looking for 1 more member to join our group lab section UI-X. Email me: minhductdn4@gmail.com

**Comment by LiamDaigle** [comment]
Hey, I am looking for a group for the project, I am in lab section UN-X. If any groups are looking for a member in that section let me know, my email is: liam.daigle@gmail.com

**Comment by YoussefMorcos** [comment]
Hey i’m looking for a group for the project i’m in lab UK-X, if you are looking for team members please contact me at youssef_samir@live.com

**Comment by Vithujanan** [comment]
Hi, looking for 1 more member. We're in lab section UI-X. You can contact me at vithujanan0629@gmail.com or reply to this thread

**Comment by Ghawi25** [comment]
Hi looking for 1 more team member. email me at alexanderghawi@gmail.com or reply

**Comment by ramy-att** [comment]
Hi, looking for 1 more member, please reply here if you need a team.

**Comment by ChemsCode** [comment]
Hey, looking for 1 more member, let me know if you need a team.
